lib/daru/io/io.rb

```diff
@@ -183,6 +183,18 @@ def load filename
         end
       end
 
+      def from_html path, opts
+        page = Mechanize.new.get(path)
+        page.search('table').map { |table| html_parse_table table }
+            .keep_if { |table| html_search table, opts[:match] }
+            .compact
+            .map { |table| html_decide_values table, opts }
+            .map { |table| html_table_to_dataframe table }
+      rescue LoadError
+        STDERR.puts '\nInstall the mechanize gem version 2.7.5 for using'\
+        ' from_html function.'
```

Comment:
I believe it should not be `puts`, but rather failing with meaningful error. Because script relying on `from_html` should be terminated if `mechanize` is not accessible, not continue with undefined results and small warning message.